GitHub_Learning/Generic/Navigation/View/WireframeImpl.swift

```diff
@@ -75,4 +77,25 @@ class WireframeImpl: Wireframe {
         vc.modalPresentationStyle = presentationStyle ?? .automatic
         window.topViewController?.present(vc, animated: animated, completion: completion)
     }
+    
+    private func makeTabBarController() -> MainAppTabBarController {
+        let dashboardVC = try! instanceProvider.getInstance(DashboardViewController.self)
+        let dashboardNavController = try! instanceProvider.getInstance(MainAppNavigationController.self)
+        dashboardNavController.setViewControllers([dashboardVC], animated: true)
+        
+        let favouritesVC = try! instanceProvider.getInstance(FavouritesViewController.self)
+        let favouritesNavController = try! instanceProvider.getInstance(MainAppNavigationController.self)
+        favouritesNavController.setViewControllers([favouritesVC], animated: true)
+        
+        let newsVC = try! instanceProvider.getInstance(NewsViewController.self)
+        let newsStubNavController = try! instanceProvider.getInstance(MainAppNavigationController.self)
+        newsStubNavController.setViewControllers([newsVC], animated: true)
+        
+        let portfolioVC = try! instanceProvider.getInstance(PortfolioViewController.self)
+        let portfolioFakeNavController = try! instanceProvider.getInstance(MainAppNavigationController.self)
+        portfolioFakeNavController.setViewControllers([portfolioVC], animated: true)
+        
+        mainAppTabBar.setViewControllers([dashboardNavController, favouritesNavController, newsStubNavController, portfolioFakeNavController], animated: false)
+        return mainAppTabBar
```

Comment:
Todle je zbytecny. Dej ty VC do initu toho tab baru. Pak ve view did load to nastavis do navigacnich controlleru a das jako view controllery.